src/Icon/Icon.jsx

```diff
@@ -30,25 +30,23 @@ const validate = name => {
   return 'M0 0L24 0L24 24 L0 24Z'
 }
 
-export const Icon = ({ name, size, color, rotate, ...props }) => (
-  <Wrapper color={color}>
-    <BaseIcon
-      path={externalGlyphs[name] || glyphs[name] || validate(name)}
-      size={sizes[size] ?? size}
-      color='currentColor'
-      spin={rotate}
-      {...props}
-    />
-  </Wrapper>
+export const Icon = ({ name, size, color, rotate, spin, ...props }) => (
+  <BaseIcon
+    path={externalGlyphs[name] || glyphs[name] || validate(name)}
+    size={sizes[size] ?? size}
+    color={color}
+    spin={spin}
+    rotate={rotate}
```

Comment:
Is the string/number distinction handled by the BaseIcon? Or we need to do:

```suggestion
    rotate={typeof rotate === 'string' ? rotate : `${rotate}deg`}
```

Reply:
The BaseIcon accepts only numbers and computes the string itself

We do not need to accept both string and numbers in our component